commitizen/config/base_config.py

```diff
@@ -10,11 +10,16 @@ def __init__(self):
         self._settings: Settings = DEFAULT_SETTINGS.copy()
         self.encoding = self.settings["encoding"]
         self._path: Path | None = None
+        self._settings_from_configs: Settings = {}
 
     @property
     def settings(self) -> Settings:
         return self._settings
 
+    @property
+    def mutated_settings(self) -> Settings:
```

Comment:
Could you add a docstring to this function explaining it a bit further? thanks
Right now it seems it does the same as `mutated_settings`, but I think the *intention* is different, right?